SpatialGDK/Source/SpatialGDK/Public/Utils/OpCallbackTemplate.h

```diff
@@ -0,0 +1,43 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#pragma once
+
+#include <WorkerSDK/improbable/c_worker.h>
+#include "OpCallbackTemplate.generated.h"
+
+UCLASS(abstract)
+class SPATIALGDK_API UOpCallbackTemplate : public UObject
+{
+	GENERATED_BODY()
+
+public:
+	UOpCallbackTemplate() = default;
+	~UOpCallbackTemplate() = default;
+
+	/**
+	  * Called by SpatialDispatcher on initialization, this is to allow the user to write more useful callbacks.
+	  * @param World the UWorld top level hierarchy object.
+	  */
+	void Init(UWorld* InWorld) {
```

Comment:
Should `Init` be `virtual` if the user wants to also set up some state in here?

Reply:
I could imagine you would maybe want to setup a database connection here or something /shrug

Reply:
I'm not sure how that's doable. It would involve users knowing that the dispatcher had instantiated the callbacks, then accessing the op callbacks map, getting the classes, then casting them to something derived that implemented the users `Init` code? Happy to expose that pipeline, if you think it's necessary, just fairly convoluted...

Reply:
resolved offline, added virtual init methods for users to do their own initialisation after World and StaticComponentView are init'd